.nf-core.yml

```diff
@@ -5,6 +5,7 @@ lint:
   - docs/images/nf-core-postprocessing_logo_light.png
   - docs/images/nf-core-postprocessing_logo_dark.png
   - .github/ISSUE_TEMPLATE/config.yml
+  - .github/workflows/branch.yml
```

Comment:
No need to do the same for the fix-linting.yml workflow?

Reply:
Lint didn't fail any tests because of that one...